cli/tox.ini

```diff
@@ -168,6 +168,20 @@ commands =
         ../util/ \
         {posargs}
 
+[testenv:pylint-pcluster3]
```

Comment:
I'd just use the generic pylint name for this section

Reply:
Done